feat(components/indicators): update icons and moment dependencies

johnhwhite opened the pull request.

blackbaud-sky-build-user commented:
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/359/storybook/)

Component Storybooks:

- [indicators-storybook](https://blackbaud.github.io/skyux-pr-preview/359/storybooks/indicators-storybook/)

johnhwhite committed:
feat(components/indicators): update icons and moment dependencies

- Added Blackbaud brandmark and download icons. [#34](https://github.com/blackbaud/skyux-icons/pull/34)
- Added show/hide icons. [#35](https://github.com/blackbaud/skyux-icons/pull/35)
- Reconciled duplicate glyphs and added new icons. [#33](https://github.com/blackbaud/skyux-icons/pull/33)
